### PR TITLE
Standardize shortcuts to start with alt key

### LINKED ISSUE #653: Standardize keyboard shortcuts
It’s too confusing to have some shortcuts start with `Ctrl` and others start with `Alt`. Given that many key combinations that start with `Ctrl` are used by the browser, we should standardize all of **em**’s to start with `Alt`.

Drop `Shift` when there is no conflict with another shortcut. e.g. `Ctrl + Shift + H` should be changed to `Alt + H`.

### PR DESCRIPTION
**Pull request by shresthabijay**
fixes #653

### CODE REVIEW

**Comment by shresthabijay** [comment]
@raineorshine I have changed most of the keyboard shortcuts to start with alt key and also dropped shift key in necessary places. I had problem updating shortcut for `Prose View`. It's current shortcut is `ctrl + shift + p`. We cannot change it to `alt+ shift + p` because `pinSubthoughts` is already using this. `ctrl + alt + p` is not working for me. So I left that shortcut as it is. Please let me know what we should do about it.

File: src/components/__tests__/Mount.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ it('create, navigate, and edit thoughts', async () => {
   await editable.simulate('change', { target: { value: 'a' } })
 
   // create subthought
-  windowEvent('keydown', { key: 'Enter', ctrlKey: true })
+  windowEvent('keydown', { key: 'Enter', altKey: true })
```
Comment (raineorshine) [inline src/components/__tests__/Mount.js:19]: All `Ctrl + Enter` and related shortcuts for creating new thoughts should use the original `ctrlKey`. Sorry I forgot about those.

File: src/shortcuts/cursorNext.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ export default {
   id: 'cursorNext',
   name: 'Cursor Next Thought',
   description: 'Move the cursor to the next thought, skipping expanded children.',
-  keyboard: { key: 'ArrowDown', meta: true },
+  keyboard: { key: 'ArrowDown', alt: true },
```
Comment (raineorshine) [inline src/shortcuts/cursorNext.js:17]: Use original

File: src/shortcuts/cursorPrev.js [hunk 3]
```diff
@@ -16,6 +16,6 @@ export default {
   description: 'Move cursor to previous thought, skipping expanded children.',
   gesture: 'lur',
   svg: Icon,
-  keyboard: { key: 'ArrowUp', meta: true },
+  keyboard: { key: 'ArrowUp', alt: true },
```
Comment (raineorshine) [inline src/shortcuts/cursorPrev.js:19]: Use original

File: src/shortcuts/delete.js [hunk 4]
```diff
@@ -80,7 +80,7 @@ export default {
   description: 'Delete the current thought.',
   gesture: 'ldl',
   svg: Icon,
-  keyboard: { key: 'Backspace', shift: true, meta: true },
+  keyboard: { key: 'Backspace', alt: true },
```
Comment (raineorshine) [inline src/shortcuts/delete.js:83]: Use original

File: src/shortcuts/home.js [hunk 5]
```diff
@@ -14,7 +14,7 @@ export default {
   id: 'home',
   name: 'Home',
   description: 'Navigate to Home.',
-  keyboard: { key: 'm', shift: true, meta: true },
+  keyboard: { key: 'm', alt: true },
```
Comment (raineorshine) [inline src/shortcuts/home.js:17]: Let's use `Alt + h` as that's a better mnemonic.

File: src/shortcuts/moveThoughtDown.js [hunk 6]
```diff
@@ -16,7 +16,7 @@ export default {
   id: 'moveThoughtDown',
   name: 'Move Thought Down',
   description: 'Move the current thought down.',
-  keyboard: { key: 'ArrowDown', meta: true, shift: true },
+  keyboard: { key: 'ArrowDown', alt: true, shift: true },
```
Comment (raineorshine) [inline src/shortcuts/moveThoughtDown.js:19]: Use original

File: src/shortcuts/moveThoughtUp.js [hunk 7]
```diff
@@ -16,7 +16,7 @@ export default {
   id: 'moveThoughtUp',
   name: 'Move Thought Up',
   description: 'Move the current thought up.',
-  keyboard: { key: 'ArrowUp', meta: true, shift: true },
+  keyboard: { key: 'ArrowUp', alt: true, shift: true },
```
Comment (raineorshine) [inline src/shortcuts/moveThoughtUp.js:19]: Use original

File: src/shortcuts/newSubthought.js [hunk 8]
```diff
@@ -20,7 +20,7 @@ export default {
   name: 'New Subthought',
   description: 'Create a new subthought in the current thought. Add it to the bottom of any existing subthoughts.',
   gesture: 'rdr',
-  keyboard: { key: 'Enter', meta: true },
+  keyboard: { key: 'Enter', alt: true },
```
Comment (raineorshine) [inline src/shortcuts/newSubthought.js:23]: Use original

File: src/shortcuts/newSubthoughtTop.js [hunk 9]
```diff
@@ -17,7 +17,7 @@ export default {
   name: 'New Subthought (top)',
   description: 'Create a new subthought in the current thought. Add it to the top of any existing subthoughts.',
   gesture: 'rdu',
-  keyboard: { key: 'Enter', shift: true, meta: true },
+  keyboard: { key: 'Enter', shift: true, alt: true },
```
Comment (raineorshine) [inline src/shortcuts/newSubthoughtTop.js:20]: Use original

File: src/shortcuts/openShortcutPopup.js [hunk 10]
```diff
@@ -12,7 +12,7 @@ export default {
   id: 'openShortcutPopup',
   name: 'Open Shortcut Popup',
   description: `Open the help screen which contains the tutorials and a list of all ${isMobile ? 'gestures' : 'keyboard shortcuts'}.`,
-  keyboard: { key: '/', meta: true },
+  keyboard: { key: '/', alt: true },
```
Comment (raineorshine) [inline src/shortcuts/openShortcutPopup.js:15]: Use original

File: src/shortcuts/toggleHiddenThoughts.js [hunk 11]
```diff
@@ -13,7 +13,7 @@ export default {
   id: 'toggleHiddenThoughts',
   name: 'Toggle Hidden Thoughts',
   description: 'Show or hide hidden thoughts.',
-  keyboard: { key: 'h', meta: true, shift: true },
+  keyboard: { key: 'h', alt: true },
```
Comment (raineorshine) [inline src/shortcuts/toggleHiddenThoughts.js:16]: Let's use `Alt + Shift + h` for this since Home will use `Alt + h`.

**Pull request merged**